bin/cli.js

```diff
@@ -1,17 +1,185 @@
 #!/usr/bin/env node
 'use strict';
 
-const metasync = require('metasync');
+const program = require('commander');
+const karma = require('karma');
+const path = require('path');
+const fs = require('fs');
 
-const walk = require('../lib/walk');
+const splitOptions = v => v.split(',');
 
-console.log('Metatest CLI Stub');
+const clOptions = [
+  [ '-c, --config <value>', 'Config file' ],
+  [ '-e, --enviroment <value>', 'Enviroment to run', 'node' ],
+  [ '-p, --required-packages <items>', 'Required packages', splitOptions, []],
+  [ '-b, --browsers <items>', 'Browsers to run', splitOptions, []],
+  [ '-p, --browser-port <n>', 'Browser port'],
+  [ '-l, --browser-log <value>', 'Browser log level']
+];
+
+const karmaLogLevels = {
+  'disable': [ 'LOG_DISABLE', 'none' ],
+  'error': [ 'LOG_ERROR', 'errors-only' ],
+  'warn': [ 'LOG_WARN', 'minimal' ],
+  'info': [ 'LOG_INFO', 'normal' ],
+  'debug': [ 'LOG_DEBUG', 'verbose' ]
+};
+
+const browserLaunchers = {
+  'Chrome': 'karma-chrome-launcher',
+  'Firefox': 'karma-firefox-launcher',
+  'IE': 'karma-ie-launcher',
+  'Opera': 'karma-opera-launcher',
+  'PhantomJS': 'karma-phantomjs-launcher',
+  'Safari': 'karma-safari-launcher'
+};
+
+const rmDuplicates = arr => Array.from(new Set(arr));
+
+const parseFile = file => JSON.parse(fs.readFileSync(path.resolve(file)));
+
+const loadFiles = files => {
+  const result = [];
+  files.forEach(file => {
```

Comment:
I think you can use `reduce` for a bit more idiomatic way, but that's fine as it is too.